app/src/main/java/com/skyyo/samples/features/textRecognition/cameraRecognition/CameraTextRecognitionScreen.kt

```diff
@@ -0,0 +1,109 @@
+package com.skyyo.samples.features.textRecognition.cameraRecognition
+
+import android.Manifest
+import android.view.View
+import androidx.camera.core.CameraSelector
+import androidx.camera.core.ImageAnalysis
+import androidx.camera.core.ImageCapture
+import androidx.camera.core.Preview
+import androidx.camera.lifecycle.ProcessCameraProvider
+import androidx.camera.view.PreviewView
+import androidx.compose.foundation.layout.systemBarsPadding
+import androidx.compose.material.Button
+import androidx.compose.material.Text
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.LocalLifecycleOwner
+import androidx.hilt.navigation.compose.hiltViewModel
+import androidx.lifecycle.compose.ExperimentalLifecycleComposeApi
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import com.google.accompanist.permissions.ExperimentalPermissionsApi
+import com.google.accompanist.permissions.PermissionRequired
+import com.google.accompanist.permissions.rememberPermissionState
+import com.skyyo.samples.extensions.goAppPermissions
+import com.skyyo.samples.extensions.tryOrNull
+import com.skyyo.samples.features.textRecognition.cameraRecognition.composables.CameraTextRecognition
+import java.util.concurrent.Executors
+
+@OptIn(
+    ExperimentalPermissionsApi::class,
+    ExperimentalLifecycleComposeApi::class
+)
+@Composable
+fun CameraTextRecognitionScreen(viewModel: CameraTextRecognitionViewModel = hiltViewModel()) {
+    val context = LocalContext.current
+    val lifecycleOwner = LocalLifecycleOwner.current
+    val recognizedSurfaceData by viewModel.recognizedSurfaceData.collectAsStateWithLifecycle()
+    val cameraPermissionState = rememberPermissionState(Manifest.permission.CAMERA)
+    val cameraExecutor = remember { Executors.newSingleThreadExecutor() }
+    val imageCapture = remember {
+        ImageCapture.Builder()
+            .setCaptureMode(ImageCapture.CAPTURE_MODE_MINIMIZE_LATENCY)
+            .build()
+    }
+    val imageAnalyzer by lazy {
+        ImageAnalysis.Builder()
+            .build()
+            .also {
+                it.setAnalyzer(
+                    cameraExecutor,
+                    TextImageAnalyzer(viewModel::onSurfaceAnalyzed)
+                )
+            }
+    }
```

Comment:
no use for `lazy` here, value should be `remembered` and placed into the `PermissionRequired.content` lambda. This will make it both "lazy" & not recreated upon recompositions.